i18n/config.ts

```diff
@@ -0,0 +1,29 @@
+import i18n from 'i18next';
+import Backend from 'i18next-xhr-backend';
```

Comment:
I'd not go would not go with this one, the backend is not ready to serve these files.

Reply:
yeah I was going to remove it :)